ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c

```diff
@@ -4092,9 +4106,16 @@ dbmdb_import_init_writer(ImportJob *job, ImportRole_t role)
 void
 dbmdb_free_import_ctx(ImportJob *job)
 {
+    WorkerQueueData_t *s = NULL;
+    int i=0;
```

Comment:
offsets to arrays are size_t. declare inside for loop as this allows further compiler optimisation.